src/Components/VideoListing/VideoListing/VideoList.jsx

```diff
@@ -0,0 +1,23 @@
+import { VideoCard } from "./VideoCard";
+import { videos } from "./../../../backend/db/videos";
+
+export const VideoList = () => {
+  return (
+    <section className="row-container">
+      {videos.map((video) => {
+        return (
+          <VideoCard
+            key={video._id}
+            thumbnail={video.thumbnail}
+            title={video.title}
+            creator={video.creator}
+            views={video.views}
+            date={video.date}
+            label={video.label}
+            glimpse={video.glimpse}
```

Comment:
Instead of this, you can pass a video as a prop and then destructured it for the cleaner JSX code.